README.md

```diff
@@ -1,17 +1,96 @@
+
+The main focus of our project is to create a desktop version for our dashboard, but we're also creating a mobile version. This way it makes it possible for bouncers to get notified on new reports, so that they can anticipate on them and help the victims when needed.
+
+In the next chapters, the features of the dashboard will be explained for both versions.
+
+## 💻 Desktop version
+
+The desktop version of the dashboard is devided in 4 parts:
+
+- Home
+- Map
+- All Reports
+- Settings
+
+### 🏠 Home
+
+On this page, multiple unique graphs and charts are being shown with data about the reports made on a particulary date/period.<br>
+Here's a screenshot of our design for the 'Home' page:<br><br>
+<img src="https://imgur.com/sErh1lt.png" width="500" height="300" style="border-radius:1em">
+
+The main function on this page is the chart that shows the amount of reports made in a week. On this chart, you can filter the weeks, and the different types of harassments reported in that week. For example, if you want to see what reports were made last week, you can just click on the button of that particular week, and it shows the amounts of reports made in that week, on each day. To see what types of harassments these reports were, you have to click on them, and it filters out the rest of the reports.
+
+Also being shown on this page is a donut chart with the different harassment types, on a particular date or period. Next to this chart is a section with a counter of the total amount of reports made on that day.
+
+These visualizations provide the most important data for club owners, and gives them the information needed with just a glance.
+
+### 📌 Map
+
+On this page, a heatmap is shown. In our case we are using a heatmap of Amsterdam, because it's one of most famous and busiest nightlife-cities of The Netherlands.
+<br>
+Here's a screenshot of our design for the 'Map' page: <br><br>
+<img src="https://imgur.com/a3q1mE1.png" width="500" height="300" style="border-radius:1em">
+
+We are using a heatmap to give an indication to the club owners to see where te most reports are made. The different club owners can then compare their clubs to others.<br>
+Because of privacy reasons, we decided to use a heatmap in stead of a regular map that pinpoints the specific locations of the reports. With this heatmap, the club owners can see the areas in stead of the exact locations of the reports. How thicker the color on the map, how more reports were made in that specific area.
+
+### 📑 All Reports
+
+On this page, all the reports are being shown in detail. Here you can see the story of the victim, the type of harassment they reported, the date and time, if the victim needs help, and if the victim wants to make contact with FemAlert. If you click on one of the reports, you can see the victims name and email adress.<br>
```

Comment:
*if the victim wants to be contacted by FemAlert
*you will see the victims name and email address, given that they gave permission for the club owner to reach out.